src/deadline/max_adaptor/MaxClient/render_handlers/__init__.py

```diff
@@ -21,5 +22,9 @@ def get_render_handler(renderer: str = "Default_Scanline_Renderer") -> DefaultMa
         return ArtHandler()
     elif renderer == "Corona":
         return CoronaHandler()
+    elif renderer == "V_Ray_6":
```

Comment:
With the removal of some of the `.strip("__")` other than checking against the allowed renderers, does the renderer not come through now as the full version string e.g. `V_Ray_6__update_#_#`? Just making sure that we don't need to be doing a `renderer.startswith()` for the VRay cases.

Reply:
I think you're right that we'll need the` .startswith() `- let me test it out locally and I'll update.

Reply:
I updated, it worked locally. I think since only V-Ray 6 is available for 3dsMax 2024 right now (and this project has that as a min. requirement) it is okay for now, but I think in the future it would be useful to add a `major_version` variable to the V-Ray handler so this if statement doesn't get too long.

Reply:
Agreed, it can be a future improvement. No sense holding this PR more on that. Approved.